src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/InnerAssignmentCheck.java

```diff
@@ -147,7 +156,9 @@
      * towards the root.
      */
     private static final int[][] ALLOWED_ASSIGNMENT_IN_COMPARISON_CONTEXT = {
-        {TokenTypes.EXPR, TokenTypes.LITERAL_WHILE, },
+            {TokenTypes.EXPR, TokenTypes.LITERAL_WHILE, },
+            {TokenTypes.EXPR, TokenTypes.FOR_CONDITION, },
```

Comment:
please remove whitespaces before right curly for 2 lines above to make it look consistent

Reply:
done